### ISSUE TITLE
t/unit/lib-Git-Lint-Check-Message/message.t may fail with non-English locale

### ISSUE THREAD

**Issue by eserte** [renderorange/Git-Lint#9]
On systems with older perl (< 5.22) and non-English locale the test suite fails:
```
#   Failed test 'exception matches expected'
#   at t/unit/lib-Git-Lint-Check-Message/message.t line 48.
#                   'open: filename: Datei oder Verzeichnis nicht gefunden at /home/cpansand/.cpan/build/2022022602/Git-Lint-0.008-zktBqU/t/unit/lib-Git-Lint-Check-Message/../../../lib/Git/Lint/Check/Message.pm line 25.
# '
#     doesn't match '(?^:open: filename: No such file or directory)'
# Looks like you failed 1 test of 7.
t/unit/lib-Git-Lint-Check-Message/message.t ..... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/7 subtests 
```
Note that it is possible to set `$!` and inspect the expected error message:
```
$ perl5.20.3 -MErrno=ENOENT -E '$!=ENOENT; say "$!"' 
Datei oder Verzeichnis nicht gefunden
$ env LC_ALL=C perl5.20.3 -MErrno=ENOENT -E '$!=ENOENT; say "$!"'
No such file or directory
```

**Comment by blainemotsinger** [comment]
Thanks, I didn't think that test assertion through.  I'll update and push a new version.

**Comment by blainemotsinger** [comment]
fixed in 0f2eca3.
